lib/screens/diagnose_screen.dart

```diff
@@ -173,16 +161,31 @@ Chat as the AI responsible to answer any follow up questions. Mention clearly if
     });
 
     try {
-      var response = await _chat.sendMessage(
-        Content.text(message),
+      setState(() {
```

Comment:
setState must not return a future (the callback cannot be async), since then it would be unclear when the state was actually being set.

Reply:
Can we move setState to after the data is received?